robosuite/environments/sawyer_lift.py

```diff
@@ -727,31 +723,51 @@ class SawyerLiftPositionTarget(SawyerLiftPosition):
     """
     def __init__(
         self,
-        object_name='cube',
-        target_name='cube_target',
         target_color=(0, 1, 0, 0.3),
         hide_target=True,
         goal_tolerance=0.05,
         goal_radius_low=0.1,
         goal_radius_high=0.2,
         **kwargs
     ):
-        self._target_name = target_name
-        self._object_name = object_name
+
         self._target_rgba = target_color
         self._goal_render_segmentation = None
         self._goal_tolerance = goal_tolerance
         self._goal_radius_low = goal_radius_low
         self._goal_radius_high = goal_radius_high
         self._goal_grid = None
-        if kwargs.get('eval_mode', False) or hide_target:
-            self._target_rgba = (0., 0., 0., 0.,)
+        self._hide_target = hide_target
+
+        self._target_name = 'cube_target'
+        self._object_name = 'cube'
+        self.interactive_objects = {}
```

Comment:
what's the point of initializing with an empty dict here?

Reply:
cuz interactive_objects is a dict?

Reply:
I meant that the whole dict can be defined where you're adding the keys